views/src/main/java/io/micronaut/views/ViewsRenderer.java

```diff
@@ -51,7 +52,20 @@
      * @param data     response body to render it with a view
      * @return A writable where the view will be written to.
      */
-    @Nonnull Writable render(@Nonnull String viewName, @Nullable Object data);
+    @Nonnull default Writable render(@Nonnull String viewName, @Nullable Object data) {
```

Comment:
Why did you change this to a default method?

Reply:
In order to not break API

Reply:
This isn't in of itself a breaking change, however it's not a good idea because it makes it easy to mistakingly change other view renderers in a breaking way. The original method should stay as it was.

Reply:
Ok. Left original method as it is and create new one with request object parameter and delegating to orig as default implementation.